### ISSUE TITLE
Incompatibility with Origins (Fabric)

### ISSUE THREAD

**Issue by pupdairy** [CodeF53/Horse-Buff#36]
I tried out this mod in my current modpack, but as soon as I launched Minecraft, I got caught in a repeating loop on the Mojang loading screen. The bar would fill almost to full, then the screen would refresh and the bar would start loading again. After some testing of my mod list, I figured out the issue was coming from how Origins (https://modrinth.com/mod/origins) was interacting with Horse Buff. I tested out the mods separately, one instance with only Horse-Buff, one with only Origins and its dependencies, and one with both. Separately, they loaded just fine, as soon as I tried to load the game with both mods at the same time, the previous issue returned. I uploaded my crash log to mclogs, hopefully something in there can be of help: https://mclo.gs/schDTyx

Even if the incompability is unfixable, at least it'll be known. Thank you for your work on the mod, it looks great and I'm sorry I couldn't enjoy like many others obviously have!

**Comment by CodeF53** [comment]
Should be fixed, try using the mod from this artifact to confirm its fixed:
https://github.com/CodeF53/Horse-Buff/actions/runs/4258771086

**Comment by CodeF53** [comment]
I don't crash when booting into a singleplayer world and choosing a origin, assumed fixed.

**Comment by pupdairy** [comment]
Yes, this is in fact fixed, thank you so much for the swift compatibility release 👍
